docs-content/general/changelog/changelog-18.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Changelog 18 (04/26)
+slug: changelog-18
+---
+
+![](https://user-images.githubusercontent.com/20292680/234690301-9fcf51a0-cf1d-4c22-b5e6-0915640bd7ec.png)
+
+## Error boundary improvements!
+We just shipped a bunch of improvements to our error boundary. Namely:
+- No longer required to import a `.css` file.
+- Lots of design updates (see above)
+- The ErrorBoundary now works seemlessly in NextJS
```

Comment:
It does work seamlessly with NextJS, but that's not new.

Also spelling on `seamless`.

Reply:
ok, delete it 👌